### DISCUSSION TITLE
"Settings option description can't be nil" message when trying to rename or create new column.

### DISCUSSION THREAD

**Discussion by rasulkireev** [community/community#46792] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

The title says it all.

In the Project settings I have the Project Name, Description and a README 
![image](https://user-images.githubusercontent.com/8257474/217667191-febedae8-801c-4604-85f7-29ca7c4a3133.png)

but when I try to change the name of the column or add a new one, I'm getting the following error:

![image](https://user-images.githubusercontent.com/8257474/217597865-9886f5fd-04ff-4b55-834b-bd4f2e90d01f.png)

Here is a short recording:
https://user-images.githubusercontent.com/8257474/217667511-639ffd48-a774-4f74-a98e-11f2aee39973.mov

**Comment by tmorgan497** [comment]
Also having this issue... 
![image](https://user-images.githubusercontent.com/21185911/217891880-9ac0812e-62cf-4bc0-a281-66c494d36df9.png)

**Reply by xoxocandykisses100** [reply]
[archive (1).zip](https://github.com/community/community/files/10934969/archive.1.zip)

**Comment by googol7** [comment]
I added a new option for the "Status" field at "Status field settings" at https://github.com/users/xxxxx/projects/1/settings/fields/Status and then it appeared as a new column.

**Reply by rasulkireev** [reply]
I'm not sure what you mean exactly, but here are my current settings:

![image](https://user-images.githubusercontent.com/8257474/218118839-367c684a-08f8-4b55-bdd8-6a54db504537.png)
![image](https://user-images.githubusercontent.com/8257474/218119003-4c6e4853-c627-4ebd-ad04-40bd2c30e771.png)
![image](https://user-images.githubusercontent.com/8257474/218119131-efe33ca6-1457-4618-bc14-1f64dfb38dfa.png)

I tried playing around with those (turning columns on and off), but the result is always the same.

**Comment by rileybroughten** [comment]
Thanks for raising this, we are working on a fix! I will post an update once this is resolved 👍

**Comment by rileybroughten** [comment, marked as answer]
A fix for this was released today, so let us know if you are still seeing any issues ✨ Thanks all for reporting!

**Reply by rasulkireev** [reply]
Yep, seems like the issue is fixed.
